tests/unit/unit1399.c

```diff
@@ -0,0 +1,96 @@
+/***************************************************************************
+ *                                  _   _ ____  _
+ *  Project                     ___| | | |  _ \| |
+ *                             / __| | | | |_) | |
+ *                            | (__| |_| |  _ <| |___
+ *                             \___|\___/|_| \_\_____|
+ *
+ * Copyright (C) 1998 - 2017, Daniel Stenberg, <daniel@haxx.se>, et al.
+ *
+ * This software is licensed as described in the file COPYING, which
+ * you should have received as part of this distribution. The terms
+ * are also available at https://curl.haxx.se/docs/copyright.html.
+ *
+ * You may opt to use, copy, modify, merge, publish, distribute and/or sell
+ * copies of the Software, and permit persons to whom the Software is
+ * furnished to do so, under the terms of the COPYING file.
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTY OF ANY
+ * KIND, either express or implied.
+ *
+ ***************************************************************************/
+#include "curlcheck.h"
+
+#include "urldata.h"
+#include "progress.h"
+
+int usec_magnitude = 1000000;
+int test_execution_time = 5000;
+int aggregate_timerids[] = {
+  TIMER_NAMELOOKUP,
+  TIMER_CONNECT,
+  TIMER_APPCONNECT,
+  TIMER_PRETRANSFER,
+  TIMER_STARTTRANSFER,
+};
+
+static bool unit_setup(void)
+{
+  return false;
+}
+
+static void unit_stop(void)
+{
+
+}
+
+static void complete_aggregate_timed_activities(struct Curl_easy *data)
+{
+  for(unsigned int i = 0; i < sizeof(aggregate_timerids); i++) {
+    Curl_pgrsTime(data, aggregate_timerids[i]);
```

Comment:
Maybe add a comment above the Curl_pgrsTime() function in progress.c to mention `@unittest: 1399`. We marked other unit-tested functions that way before and it could be worth keeping that tradition (to help us slowly test more and more functions).